src/Response/Response.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Visiosoft\GlobalHelperExtension\Response;
+
+class Response
```

Comment:
I can not understand this global helper extension.

what is the goal of this class?

what is your idea to write it as a treat class? why you prefer do not to use treat class?

can you show the functionality by an example for success true and false?

does it cover the previous response format that we used in projects?

Would you be able to give me an error response? 

how do you handle the error response and message?

what will happen when app debug is true?

where is the response status code?

I think it does not cover necessary requirements of the restful API concept. for example

for CREATE (POST) request response we have to set response status 201.
for PUT response, we have to set response code 200.
for DELETE request response:
```
A successful response of DELETE requests SHOULD be an HTTP response code 200 (OK) if the response includes an entity describing the status.
The status should be 202 (Accepted) if the action has been queued.
The status should be 204 (No Content) if the action has been performed but the response does not include an entity.
Repeatedly calling DELETE API on that resource will not change the outcome – however, calling DELETE on a resource a second time will return a 404 (NOT FOUND) since it was already removed.
```


I recommend adding an error handler function, declaring sendSuccess and sendError response separately because we can use a logs service for that.

Reply:
The main task of this class is to centralize the format between APIs or controllers. You can use this both within the controller, between functions and in restfull services. You can create a threat for your api and use it in the threat. 
You can get this format with this helper.
`{"success":true,"message":"Related Message", "data:related data"}`
`{"success":false,"message":"Related Message", "data:related data"}`

Reply:
Also. This is a global class you can develop for your needs.